contracts/modules/governance/MemberRoles.sol

```diff
@@ -222,6 +229,15 @@ contract MemberRoles is IMemberRoles, Governed, MasterAwareV2 {
       tokenController().isStakingPoolManager(msg.sender) == false,
       "MemberRoles: Member is a staking pool manager"
     );
+    
+    IPooledStaking _legacyPooledStaking = legacyPooledStaking();
+
+    // check that there are no NXM tokens left to withdraw in Nexus V1
+    require(_legacyPooledStaking.stakerDeposit(msg.sender) == 0, "V1 stakerDeposit != 0");
+    require(_legacyPooledStaking.stakerReward(msg.sender) == 0, "V1 stakerReward != 0");
+    require(_tokenController.tokensLocked(msg.sender, "CLA") == 0, "V1 CLA tokensLocked != 0");
+    (, , uint coverNotesAmount) = _tokenController.getWithdrawableCoverNotes(msg.sender);
+    require(coverNotesAmount == 0, "V1 coverNotesAmount != 0");
```

Comment:
there's another way to do this where
you have 1 require that checks the sum of all 4 is 0
_legacyPooledStaking.stakerDeposit(msg.sender) + tokenController.tokensLocked(msg.sender, "CLA") +...

It's less code, looks more elegant

personally i prefer this explicit approach

Reply:
thought about a function that does all these checks since only the address changes.

It's unconventional though about how we write code, we usually leave require statements explicit at the begining and a function that's all require statements looks like a modifier